src/parser/flex_lexer.h

```diff
@@ -92,13 +92,25 @@ typedef unsigned int flex_uint32_t;
 
 #endif /* ! FLEXINT_H */
 
-/* TODO: this is always defined, so inline it */
-#define yyconst const
+#ifdef __cplusplus
```

Comment:
@javrucebo do you have any idea why these changes appeared? Different version of flex?

Reply:
probably yes. I had previously not looked into any changes of the generated files by bison/flex and only commited the changes then when I have seen that the CI process tests both with the check-in version and with the re-generated one. obviously the first check then failed when not committing the changes to the generated bison/flex files